examples/00_quick_start/sar_movieratings_with_azureml_designer.ipynb

```diff
@@ -10,9 +10,9 @@
     "\n",
     "[AzureML Designer](https://docs.microsoft.com/en-us/azure/machine-learning/concept-designer) lets you visually connect datasets and modules on an interactive canvas to create machine learning models. \n",
     "\n",
-    "![img](https://recodatasets.blob.core.windows.net/images/designer-drag-and-drop.gif)\n",
+    "One of the features of AzureML Designer is that it is possible for developers to integrate any python library to make it available as a module/component. In this notebook are are going to show how to integrate [SAR](sar_movielens.ipynb) and several other modules in Designer.\n",
```

Comment:
we are not testing it, in fact it won't be easy to do it.

We actually considered to make it a MD file, but rejected the idea because there is an execution that is done in the notebook to load the system in Designer